changelogs/fragments/snmp_facts.yml

```
@@ -0,0 +1,2 @@
+security_fixes:
+- **SECURITY** - CVE-2021-20178 - hide user sensitive information such as ``privkey`` and ``authkey`` from logging into the console.
```

This should fix the yaml error
```suggestion
- '**SECURITY** - CVE-2021-20178 - hide user sensitive information such as ``privkey`` and ``authkey`` from logging into the console.'
```